README.md

```diff
@@ -43,6 +43,7 @@
 <div align="center">
     <sub>Alternative sites for the project (unofficials)</sub>
     <br />
+    <a href="https://www.collective-api.ml">Collective APIs</a> •
```

Comment:
You must add this link after Apihouse.

Ex:

```markdown
(...)
    <a href="https://devresourc.es/tools-and-utilities/public-apis">Dev Resources</a> •
    <a href="https://www.public-apis.ml">Public APIs Site</a> •
    <a href="https://apihouse.vercel.app">Apihouse</a> •
    <a href="https://www.collective-api.ml">Collective APIs</a>
</div>
```

Reply:
Hey @matheusfelipeog, thanks for reviewing my project. I have resolved your suggestion: https://github.com/public-apis/public-apis/pull/2068/commits/3f0a7ee29a502354ba6f99d3b648160df8056839 ❤